internal/dockerfile/docker.go

```diff
@@ -94,7 +94,7 @@ RUN apk add --no-cache --no-progress gcc git make musl-dev
 
 COPY . /src
 ARG BININFO_BUILD_DATE BININFO_COMMIT_HASH BININFO_VERSION # provided to 'make install'
-RUN make -C /src install PREFIX=/pkg%[4]s
+RUN make -C /src install PREFIX=/pkg GOTOOLCHAIN=local%[4]s
```

Comment:
Maybe we should even write that into the makefile that devs don't accidentally download some version and test with it?